### ISSUE TITLE
Regression in 0.26 StringEnum example

### ISSUE THREAD

**Issue by daffl** [sinclairzx81/typebox#364]
The following schema with the `StringEnum` from the [Unsafe documentation](https://github.com/sinclairzx81/typebox#unsafe):

```ts
function StringEnum<T extends string[]>(values: [...T]) {
  return Type.Unsafe<T[number]>({ type: 'string', enum: values })
}

const testSchema = Type.Partial(
  Type.Object({
    names: StringEnum(['A', 'B'])
  })
)
```

Produces this JSON schema in 0.25.24:

```json
{
  "type": "object",
  "properties": {
    "names": {
      "type": "string",
      "enum": [
        "A",
        "B"
      ]
    }
  }
}
```

And this JSON schema in 0.26.7:

```json
{
  "type": "object",
  "properties": {
    "names": {
      "type": "string",
      "enum": [
        "A",
        "B"
      ]
    }
  },
  "required": [
    "names"
  ]
}
```

Note that the schema should not have any `required` fields.

**Comment by sinclairzx81** [comment]
@daffl Fix should be ready on `0.26.8`.

**Comment by daffl** [comment]
Verified that this is working now, thank you for the quick fix!
